src/doc/en/developer/coding_basics.rst

```diff
@@ -153,17 +153,53 @@ Adding new top-level packages below :mod:`sage` should be done
 sparingly.  It is often better to create subpackages of existing
 packages.
 
-Non-Python Sage source code and supporting files can be included in one
-of the following places:
+Non-Python Sage source code and small supporting files can be
+included in one of the following places:
 
 - In the directory of the Python code that uses that file.  When the
   Sage library is installed, the file will be installed in the same
-  location as the Python code. For example,
-  ``SAGE_ROOT/src/sage/interfaces/maxima.py`` needs to use the file
-  ``SAGE_ROOT/src/sage/interfaces/maxima.lisp`` at runtime, so it refers
-  to it as ::
+  location as the Python code. This is referred to as "package data".
 
-    os.path.join(os.path.dirname(__file__), 'sage-maxima.lisp')
+  The preferred way to access the data from Python is using the
+  `importlib.resources API <https://importlib-resources.readthedocs.io/en/latest/using.html>`,
+  in particular the function :func:`importlib.resources.files`.
+  It should be imported as follows
+  (see :ref:`section-python-language-standard`)::
+
+    import sys
+
+    if sys.version_info < (3, 11):
+        # Use backport package providing Python 3.11 features
+        from importlib_resources import files
+    else:
+        from importlib.resources import files
```

Comment:
```suggestion
    from importlib.resources import files
```
I already gave rationale.

Reply:
If you don't agree with the suggestion, feel free to label the PR as disputed.

Reply:
That's not how it works.

Reply:
I have taken this part out now to regain velocity

Reply:
Thanks.